nss-day-cohort-60/rare-client-tenacious-tators
# view comments by postId, link on PostList, button on PostDetails

Closes #5 
Closes #57 


# Description of Proposed Changes

Added Link to ../posts -- Post Title navigates to ../posts/id
Added View Comments button to Post Details
When clicked you are navigated to page that displays only comments with the same post ID
---


# Steps to Test

1. Fetch the [jes-view-post-comments-client](https://github.com/nss-day-cohort-60/rare-client-tenacious-tators/compare/main...jes-view-post-comments-client?expand=1) branch and switch to it
2. have server open and running, if you have no comments in your comments table, run the following insert commands to add some:
3. 
INSERT INTO Comments ('post_id', 'author_id', 'content') VALUES (1, 1, 'So good!');
INSERT INTO Comments ('post_id', 'author_id', 'content') VALUES (1, 2, 'The best content on the web!');
INSERT INTO Comments ('post_id', 'author_id', 'content') VALUES (2, 1, 'Very enlightening.');
INSERT INTO Comments ('post_id', 'author_id', 'content') VALUES (2, 2, 'So glad I came across this article!');

4. login to Rare Publishing with a username and password from your local SQL db
5. navigate to Posts, click a post title and verify you are navigated to correct post details page
6. while on post details page, click View Comments button and verify you are viewing only the comments for that post (for testing I've only included comments for post 1 and 2--if you want to verify more postIds you'll need to add more comments to your SQL db)
